specs/eip4844/polynomial-commitments.md

```diff
@@ -449,21 +495,24 @@ def compute_aggregate_kzg_proof(blobs: Sequence[Blob]) -> KZGProof:
 
 ```python
 def verify_aggregate_kzg_proof(blobs: Sequence[Blob],
-                               expected_kzg_commitments: Sequence[KZGCommitment],
-                               kzg_aggregated_proof: KZGProof) -> bool:
+                               expected_kzg_commitments_bytes: Sequence[Bytes48],
+                               kzg_aggregated_proof_bytes: Bytes48) -> bool:
```

Comment:
For consistency, could we rename these to `expected_commitments_bytes` & `aggregated_proof_bytes`, respectively?

Reply:
Done in d6bb1b9ff